src/Illuminate/Support/helpers.php

```diff
@@ -883,3 +883,15 @@ function with($object)
         return $object;
     }
 }
+
+if (! function_exists('is_winnt')) {
+    /**
+     * Determine whether the current envrionment is Windows NT based.
+     *
+     * @return bool
+     */
+    function is_winnt()
+    {
+        return substr(strtoupper(php_uname('s')), 0, 3) === 'WIN';
```

Comment:
@vlakoff That is a necessary, but not sufficient condition, thus cannot be used.

Windows => \ seperator
BUT
\ seperator !=> Windows